reference/configuration/twig.rst

```diff
@@ -82,6 +82,12 @@ is not recommended; not even in the ``dev`` environment, because the
 ``auto_reload`` option ensures that cached templates which have changed get
 compiled again.
 
+Set this option to the path where the cache should be stored. If set to
+``true``, the cache is stored in ``%kernel.cache_dir%/twig``. However, if the
+``auto_reload`` option is not enabled and ``%kernel.build_dir`` is configured
+to be different than ``%kernel.cache_dir``, the cache is stored in
+``%kernel.build_dir%/twig``.
```

Comment:
I've rewritten this paragraph trying to simplify the wording and logic.
"not enabled" is "disabled"

```suggestion
Specify the path where the cache should be stored. If set to `true`, the cache
defaults to `%kernel.cache_dir%/twig`. However, if the `auto_reload` option is
disabled and `%kernel.build_dir%` is different from `%kernel.cache_dir%`,
the cache will instead be stored in `%kernel.build_dir%/twig`.
```